CHALLENGE 2.0/def,trian_calc.py

```diff
@@ -0,0 +1,14 @@
+def trian_calc ():
+    A=float(input("insert a number here; "))
+    B=float(input("insert a number here; "))
+    C=float(input("insert a number here; "))
```

Comment:
You've not removed the input function. Do something like this and use the input function outside this triangle function
```suggestion
def triangle_calculation(side1, side2, side3):
   if  side1+side2 > side3 and  side1+side3 > side2 and side2+side3 > side1
   ...
   ```